agent/engine/docker_task_engine_test.go

```diff
@@ -2274,6 +2274,8 @@ func TestSynchronizeResource(t *testing.T) {
 	cgroupResource.EXPECT().TerminalStatus().MaxTimes(1)
 	cgroupResource.EXPECT().SteadyState().MaxTimes(1)
 	cgroupResource.EXPECT().GetKnownStatus().MaxTimes(1)
+	cgroupResource.EXPECT().GetName().AnyTimes().Return("cgroup")
+	cgroupResource.EXPECT().StatusString(gomock.Any()).AnyTimes()
```

Comment:
Should the mock method be called `String()`?

Reply:
i think the method is called StatusString in the task resource interface: https://github.com/aws/amazon-ecs-agent/blob/master/agent/taskresource/interface.go